.buildkite/rllib.rayci.yml

```diff
@@ -106,24 +93,43 @@ steps:
       - bazel run //ci/ray_ci:test_in_docker -- //rllib/... rllib
         --workers "$${BUILDKITE_PARALLEL_JOB_COUNT}" --worker-id "$${BUILDKITE_PARALLEL_JOB}"
         --only-tags gpu
+        --except-tags multi_gpu,manual
         --test-env=RAY_USE_MULTIPROCESSING_CPU_COUNT=1
         --test-env=RLLIB_NUM_GPUS=1
         --build-name rllibgpubuild-py3.10
         --python-version 3.10
     depends_on: rllibgpubuild
 
+  - label: ":brain: rllib: multi-gpu tests"
+    tags:
+      - rllib_gpu
+      - gpu
+      - skip-on-microcheck
+    parallelism: 5
+    instance_type: gpu-large
+    commands:
+      - bazel run //ci/ray_ci:test_in_docker -- //rllib/... rllib
+        --workers "$${BUILDKITE_PARALLEL_JOB_COUNT}" --worker-id "$${BUILDKITE_PARALLEL_JOB}"
+        --parallelism-per-worker 2
+        --gpus 4
+        --only-tags multi_gpu
+        --except-tags manual
+        --build-name rllibgpubuild-py3.10
+        --python-version 3.10
+    depends_on: rllibgpubuild
+
   - label: ":brain: rllib: data tests"
     tags:
       - data
       - rllib
-      - disabled  # Tests of this tag do not exist any more.
+      - disabled  # Tests of this tag do not exist anymore.
     instance_type: large
     commands:
       # learning tests pytorch
       - bazel run //ci/ray_ci:test_in_docker -- //rllib/... rllib
```

Comment:
Let't keep an eye on this `bazel run ...`, subject to discussion with Simon re:  `learning_tests_with_ray_data`